sw/host/opentitanlib/BUILD

```diff
@@ -304,6 +308,7 @@ rust_test(
         "src/otp/testdata/otp_ctrl_img_dev.hjson",
         "src/otp/testdata/otp_ctrl_mmap.hjson",
         "src/otp/testdata/output.vmem",
+        "//third_party/openocd:jtag_cmsis_dap_adapter_cfg",
```

Comment:
Do you need to specify the data here as well? It's already specified in `:opentitanlib` I think.

Reply:
I would think the same, and at first I had not specified, but then bazel complained.

You can even see how the CI complains that the documentation rule does not also include this same data.  I think that there is something that I am not doing right.

`in rust_doc rule //sw/host/opentitanlib:opentitanlib_doc: label '//third_party/openocd:jtag_cmsis_dap_adapter_cfg' in $(location) expression is not a declared prerequisite of this rule`
https://dev.azure.com/lowrisc/opentitan/_build/results?buildId=129009&view=logs&jobId=3a559e2a-952e-58d2-b8db-2e604a9266d7&j=3a559e2a-952e-58d2-b8db-2e604a9266d7&t=5d859c63-56da-5106-2237-dc76198705d5